src/examples/simple-build-and-push.yml

```diff
@@ -36,8 +36,8 @@ usage:
             # set this to true to create the repository if it does not already exist, defaults to "false"
             create-repo: true
 
-            # ECR image tag, defaults to "latest"
-            tag: myECRRepoTag
+            # ECR image tags (comma separated string), defaults to "latest"
+            tag: latest,myECRRepoTag
```

Comment:
I don't think this would work because the push-image command doesn't work as expected with a comma separated string of tags?

Reply:
I've now added support for comma separated tags in push-image command :)